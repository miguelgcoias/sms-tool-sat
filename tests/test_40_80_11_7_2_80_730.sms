40
70 9 88 5 1 3 1 2 2
59 9 77 4 4 2 2 1
50 5 55 3 2 1 2
74 4 82 2 2 2
50 10 70 3 2 4 4
58 6 70 6 1 1 1 1 1 1
9 5 14 1 5
47 6 53 1 6
26 8 34 6 1 1 1 2 1 2
70 8 78 4 2 1 2 3
60 5 65 5 1 1 1 1 1
13 6 19 6 1 1 1 1 1 1
44 2 48 2 1 1
24 9 33 1 9
0 1 1 1 1
12 3 18 1 3
55 5 65 3 1 2 2
62 3 68 2 1 2
28 4 32 2 1 3
15 10 35 6 1 2 2 2 2 1
28 5 38 2 3 2
53 6 65 5 1 1 1 2 1
62 9 71 5 1 2 2 3 1
80 9 98 5 2 1 2 3 1
67 5 77 4 1 1 1 2
0 3 6 3 1 1 1
61 5 66 5 1 1 1 1 1
44 10 54 1 10
13 9 31 6 1 2 1 1 1 3
31 2 33 2 1 1
55 7 62 5 2 1 2 1 1
32 6 44 4 1 1 2 2
57 2 59 2 1 1
55 5 65 4 1 1 1 2
39 7 46 4 2 1 2 2
3 9 12 3 2 2 5
34 6 40 5 1 1 1 2 1
80 3 86 1 3
23 7 30 5 1 2 2 1 1
61 3 67 3 1 1 1
2 26 7
0
1 21
3 39 7 25
1 28
1 12
0
1 36
0
1 18
0
0
2 20 21
1 12
0
1 7
4 21 16 20 7
0
0
2 12 15
0
1 9
1 36
3 25 28 22
4 8 27 22 13
0
0
1 14
0
0
0
0
1 28
2 22 28
0
0
1 12
4 28 22 8 12
1 26
1 19
