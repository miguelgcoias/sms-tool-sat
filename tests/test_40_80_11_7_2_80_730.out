14
1 78 79 82 83 85
3 50 52 53
7 9
10 70 72 73 75
15 0
16 15
18 65 66
19 28 29
21 32 35
26 1 2 3
29 14 18 20 21 22 23
31 55 57 58 60 61
32 37 38 39 41
40 62 63 64
